wfdb/io/record.py

```diff
@@ -6,16 +6,11 @@
 import numpy as np
 import os
 import pandas as pd
-import math
-import functools
-import struct
-import pdb
 
 from wfdb.io import _header
 from wfdb.io import _signal
 from wfdb.io import _url
 from wfdb.io import download
-from wfdb.io import annotation
```

Comment:
This looks like it used to be a circular import, and I don't know how it was working. See how `annotation.py` imports `record.py`. This was only here because of the `sigavg` function which I moved to the more appropriate `processing` subpackage.

Reply:
I don't quite understand how Python accepts/deals with cyclic deps... https://stackoverflow.com/questions/7336802/how-to-avoid-circular-imports-in-python